classes/module/Module.php

```diff
@@ -1146,20 +1149,18 @@ protected static function coreLoadModule($module_name)
      */
     public static function getInstanceById($id_module)
     {
-        static $id2name = null;
-
-        if (is_null($id2name)) {
-            $id2name = array();
+        if (is_null(self::$id2name)) {
```

Comment:
You usually ask for `null === self::$id2name`, why don't you make this test?

![Y U NO](https://i.kym-cdn.com/entries/icons/original/000/004/006/YUNO.jpg)

Reply:
WASN'T me :(

Reply:
But I changed it!